### ISSUE TITLE
Rename the file after upload will sometimes remove the file type icon

### ISSUE THREAD

**Issue by codejamtc** [appirio-tech/connect-app#3840]
### Expected behavior
Better to remove the file extension from the title section so that user won't accidentally remove it while renaming

### Actual behavior
Rename the file after upload will sometimes remove the file type icon

### Steps to reproduce the problem
- Login as Customer
- Click on a project that has permission
- Click "Assets Library"
- Upload a file (Word.doc) and select 'Member X' from the 'OR ONLY SPECIFIC PEOPLE' section > Click 'Share with selected members
- Now Edit the file and change the name Word.doc >> Word Edited
- Click 'Save Changes'
- Check the file on the list

### Screenshot/screencast

![image](https://user-images.githubusercontent.com/42398485/78295726-34810600-74f2-11ea-9bf8-d249b45221d6.png)

#### Environment
- OS: Window 10
- Browser (w/version): Chrome 80.0.3987.149 (Official Build) (64-bit)
- User role (client, copilot or manager): Customer
- Account used:  CustomerUser

**Comment by vikasrohit** [comment]
@codejamtc I think it is happening because you have removed the extension while renaming the file.

**Comment by codejamtc** [comment]
@vikasrohit Yes, but people may not pay attention to the extension while renaming. Only a suggestion

**Comment by vikasrohit** [comment]
Agreed. But I think it is same behaviour in Mac OS as well. Only possible enhancement can be to alert user when he/she is changing the extension which I think is good but not a priority thing for us as we are not building File system. But We can keep it as enhancement request for future work.

**Comment by maxceem** [comment]
@vikasrohit should we implement this enhancement?

**Comment by vikasrohit** [comment]
No. I don't think it as priority stuff. Moving to backlog.
